installer/mac/pg/src/makefile

```diff
@@ -82,15 +75,16 @@ $(SHAREDPREFIX)/lib/libssl.dylib: openssl-${OPENSSL_VERSION}/Makefile
 openssl-${OPENSSL_VERSION}/Makefile: openssl-${OPENSSL_VERSION}/Configure
 	cd openssl-${OPENSSL_VERSION} && ./Configure --prefix="${SHAREDPREFIX}" darwin64-x86_64-cc zlib no-asm no-krb5 shared
 
-openssl-${OPENSSL_VERSION}/Configure: openssl-${OPENSSL_VERSION}.tar.gz
-	$(TAR) openssl-${OPENSSL_VERSION}.tar.gz
+openssl-${OPENSSL_VERSION}/Configure: openssl-${OPENSSL_VERSION}.tar.gz-${OPENSSL_SHA256}
+	$(TAR) openssl-${OPENSSL_VERSION}.tar.gz-${OPENSSL_SHA256}
 	touch $@
 
-openssl-${OPENSSL_VERSION}.tar.gz:
+openssl-${OPENSSL_VERSION}.tar.gz-${OPENSSL_SHA256}:
 	$(CURL) "https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz"
+	ruby ../../tools/renamesha256.rb openssl-${OPENSSL_VERSION}.tar.gz
```

Comment:
Feel free to use `$CHAIN` here instead of relative paths, that might be a bit more practical. (It's robust when copy-pasting code from this file to another place.) So for example

```
$CHAIN/bin/renamesha256 openssl-${OPENSSL_VERSION}.tar.gz
```